src/sass/table/stories/table.stories.js

```diff
@@ -0,0 +1,263 @@
+export default { title: "Skin/Table" };
+
+export const base = () => `
+<div class="table">
+    <table>
+        <caption>
```

Comment:
Do we have any guidance on caption styling and position? In most cases we see the table caption being programmatically associated with a nearby h2 element. Should we cover that case also, or instead of?

Reply:
@ianmcburnie , DS didn't mention anything about captions. We don't have visual specs for it. It's one of the things we need for future iterations. I included them here to account for them and require specs. I was debating whether to clip them or not by default, but I think we should push for the visual use as the default, and clipping as an option.

Sure, we can have an example like that. How would I associate the `caption` with a header?

Reply:
I'm not a fan of clipping the caption, because then we have a mismatch between visual UI and a11y tree.

I also don't think what we have currently here looks good with the caption appearing unsettled in top left.

How about we treat this similar to textbox/checkbox etc in Skin, whereby all labelling/captioning of the table is left to the app developer?

Or we could have a section after "Default Table" called "Captioned Table" showing the two ways to caption a table. Then ommit it from all other examples.

You can associate a table with a header by using `aria-describedby` on the table tag.

Reply:
Given that this is alpha, does not yet have an ebay ui piece, and is going to be quickly iterated on right after, the visual oddity is not that big of a deal IMO. That said, maybe we shouldn't worry about any of it?

I was under the impression that `caption` was critical. If that's not the case, given that DS doesn't have any specs for it at all, can we just remove it entirely? If/when we have specs, we can roll it out separately.

Reply:
Yes, I'd err on the side of just removing it for now. We could maybe add a short paragraph explaining that a table can be described with a caption element or nearby heading.

Reply:
Still feels odd not to have it; It feels like an `a11y` that we typically try to avoid in our modules. Ok, I'll remove it. If we're removing it, I'd rather not mention it at all for now. If a team uses it, they will expose the same visual oddity that we're trying to avoid here and the reason why we're removing it ourselves.

Reply:
Fixed. Resolving.